core/templates/site/searchPage.gohtml

```diff
@@ -1,27 +1,11 @@
-{{ define "searchPage" }}
     {{ template "head" $ }}
-    {{ if .AnySearch }}
     <form method="post">
```

Comment:
@jules if no access we shouldn't show them the search at all. Just tell them that search is currently disabled